byterun/misc.c

```diff
@@ -63,21 +63,21 @@ void caml_gc_message (int level, char *msg, uintnat arg)
 CAMLexport void caml_fatal_error (char *msg)
 {
   fprintf (stderr, "%s", msg);
-  exit(2);
+  abort();
```

Comment:
I don't know anything about `exit(n)` vs. `abort`. Could you comment on the potential impact of this change on users? As far as I can tell, this is the only part of the patch that is invasive, in the sense that it modifies the behavior for non-afl users.